include/rmm/device_vector.hpp

```diff
@@ -19,6 +19,9 @@
 #include <rmm/mr/device/thrust_allocator_adaptor.hpp>
 
 #include <thrust/device_vector.h>
+#ifdef THRUST_WRAPPED_NAMESPACE
+using namespace THRUST_WRAPPED_NAMESPACE;
+#endif
```

Comment:
For simplicity and tidiness, could these 3 lines just be put in a header that all RMM headers include?

Reply:
How did you choose which headers to put this in? Is it needed in the ones that do not use Thrust like `cuda_async_memory_resource.hpp`?

If it is needed everywhere, you could put it in the base MR header (`device_memory_resource.hpp`) then all MR headers would include it.

Reply:
This logic only needs to be placed in headers that explicitly `include` thrust headers.

Given that rmm has 8 headers that include thrust, refactoring to a common header seems wise.

Reply:
I have updated the pr to move the shared logic to `thrust_namespace.h`